src/Sarif.Viewer.VisualStudio.Interop/SarifViewerInterop.cs

```diff
@@ -111,7 +111,7 @@ public Task<bool> OpenSarifLogAsync(string path)
 
             return this.CallServiceApiAsync(ViewerLoadServiceInterfaceName, (service) =>
             {
-                service.LoadSarifLogs(path);
+                service.LoadSarifLog(path);
```

Comment:
>LoadSarifLog [](start = 24, length = 12)

we're going to shortly need an API that accepts an in-memory SARIF log (because we won't want to require that a background checker persist logs to disk, for perf reasons). Does this API exist currently?

Reply:
It doesn't, but now's a fine time to add it (next PR).

---
In reply to: [514521066](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514521066) [](ancestors = 514521066)